### ISSUE TITLE
A new player's experience, and a request for change.

### ISSUE THREAD

**Issue by keybounce** [DarkPacks/SevTech-Ages#2977]
## Context

I am trying to give feedback on what the pack is like for a new player, who has not played it before. I suspect that you have gotten so familiar with the pack that you don't even think about the experience of a new player.

The most important player you have is the new player. If you do not get new players, you wind up with a declining population and eventual nothingness; additionally, new players recruit other new players.

As a new player, this is what I have seen, that really seems to define the pack for me:

Go to a kitchen counter (you can do this as a thought experiment if you want), and clear a meter square surface. Use some tissue paper (or toilet paper), and divide it into 1' x 1' squares. Stand in front of it. Now, keeping your eyes straightforward, move your neck to where you are looking around at all 9 spaces. Notice how much head movement you have, notice how much work is involved.

Now, just look at the whole table, and move your hand around all 9 spaces. Notice how much easier it is?

This is without taking into account that the mouse movements needed to aim where you are looking at for the 3 spaces closest to you are actually much more exaggerated than for the others.

An "in-world crafting table", or "non-GUI interface" is not "more realistic". It is denying the use of hands to do things. It is taking the worst choices of the interface and controls of Minecraft and magnifying their effect. And it completely eliminates the concept of saying, "I'm focusing on the crafting table, I'm using my hands to place things on the table, and I'm going to look things up in the recipe book".

Good or bad, you have basically chosen to document the modpack and its recipes with JEI. And that documentation is lost as soon as you are using the work stump.

The mod page for primal tech (work stump mod) claims that it has JEI support. Yet I don't see it in play.

Now, let's look at these: work stump, wooden hopper, chest, shelf (cupboard), water strainer. The basic operations on stores of inventory items are deposit one, deposit stack, remove stack, remove one, remove half stack. The documentation for what key/mouse commands do what is nonexistent. Each block seems to have a slightly different set of controls than the other blocks, and none of them are documented. If you make a mistake with the water strainer (taking out too much), you can't put the others back. If you try to take one repeatedly from either the wooden hopper, the chest, or the shelf you wind up placing the block that you just took out (because the control used is shift-place). Etc. (and why does the water strainer have an inventory GUI when nothing else at this tech level does?).

"Nonintuitive" only begins to describe the pain of this system.

I haven't even gotten to the strangeness of saying that there is an item you can craft at tech level tutorial that includes an item you can't understand, that can be made "in hand" in your 2 x 2 but not on the work stump.

All of these "in world" non-GUI blocks essentially restrict you to only those items that you have room for in your hot bar. None of the rest of your inventory space is usable. My hot bar is almost always nearly full in normal play, and even so far in the pack I don't have much space in it.

I want to bring up the issue of the chopping block again. I complained about the behavior of the chopping block. I was told, "You tap attack on the chopping block..... stop wasting our time with these issues use discord....". Nowhere in the documentation that I saw -- not the pages for the mods, not any instruction in the achievement tree, no tooltips for the chopping block, nothing -- is the "tap attack" mentioned. And this is so much in line with everything else in Minecraft, that you see this tap attack used ... to pick up mine carts. And I think only to pick up mine carts. It isn't the norm, I am used to other mods that distinguish between your activities based on where you are pointing at -- heck, vanilla will rely on where you are aiming when it comes to placing stairs or slabs -- and to have my "bug report" (and the behavior did seem very buggy and exceedingly nonobvious) dismissed and closed with that remark seems very much "anti-new player".

And "stop wasting our time"? Do you have any idea how much time I have been spending trying to find something worth playing in this pack? Do you have any idea how much time I'm spending reporting the problems and flaws I'm running into as a new player? Do you have any idea how much time anyone else that tries to pick up this pack is going to spend trying to deal with the interface strangeness?

My observation so far is that while there appeared to be a very large amount of things to do in tech level tutorial, the reality is that the arrangement of the achievements is highly misleading, and tech level tutorial is really tiny; most of it is actually tech level 0. But even in zero, while there appears to be many things to do, all that really matters is the "advanced to the next tech level" path.

Let me repeat that: I am being trained, early in the game, that all that matters is advancing to the next tech level, and everything else at any given stage of the game is pointless.

You have gone to a great deal of effort to make these tech levels, put content in them, and all I am learning is to skip them as quickly as possible. That's just bad.

I like the idea of a detailed progression pack, complete with an actually complete progression chart. The last time I saw something like this was Jaded's packs. (My only exposure to non-vanilla skyblock was agrarian skies, and an entire section of her quest lines was a chapter on how to skyblock.). I have tried playing other quest packs that did not actually have enough information in the quest lines that you'd have any idea what you were actually trying to do or how to do it. I really do appreciate the large amount of effort (and having tried to make a quest pack once myself, I do mean that I recognize and appreciate the large amount of effort). I have been told by my friends -- both the one who encouraged the rest of us to play this, and the other one that has gone crazy with the musical ceremonies -- that the pack gets much more interesting once I get to the musical ceremonies. I've even discovered that there is a bit of balance issue -- a single person trying to generate enough music by themselves will have trouble on some of the more complicated ceremonies, but 4 people playing music together will find that all of them are trivial. I like the idea of being required to learn mods that I normally do not use -- because there actually is an in game training for these mods, which is usually the point missing from most mods.

But if there is no reason to stay around at a given tech level, if the goal that I am learning is "just advance to the next tech level, and get access to better, more advanced and more powerful mods", then why?

Would I recommend this pack to anybody else? Not based on what I've seen so far.  
Why am I trying to continue playing this pack? Only because I have 3 friends at tech level 2, and they are waiting for me to catch up so we can play together. And I'm going to ask them to put a standard chest, and a standard crafting table in my room so that I can at least reduce the amount of pain that I have while playing (and I'm going to take advantage of the fact that they already have a horse powered chopping block set up so that I never have to worry about that tap attack issue again).

Or, you know, just rely on the fact that I have a water strainer to give me all the planks I need. How does that make any sense anyways?

**Comment by artdude543** [comment]
I'll go over this in detail later on. But to clarify one thing until then.

> "You tap attack on the chopping block..... stop wasting our time with these issues use discord...."

This was posted due to the fact you opened a tonne of petty issues on that day all of which lack details and you expected us to help/give support on little to now information. That and most of them are _Discord_ issues. This tracker is for bug reports and mod suggestions not.

> I hate this or this is silly change this etc...

This is why we have Discord it's open and more people can make an input on helping/teaching people. This is a hobby for myself and all the developers who make this pack. We're not going to waste our own time when working on the pack and then deal with issues which no effort was put into them or could have been dealt with in Disord. When we have pack testers, moderators and even the user base helping each other out. If you felt hurt by that then you need to understand that we're not going to spoon feed users on this tracker on how to do x or y.

A lot of modpacks use new mods or mods which change mechanics which do lack documentation. But this is why those packs have Reddit's or Discord or even Forums where the community creates the documentation.

One big example being this https://borg286.github.io/sevtech/ like this was all done by the community. It's linked in Discord on our Wiki and even on top results for Google. But that's all I'm going to cover for the time being. I'll comment in regards to each section later but I wanted to hit this on the nail.

**Comment by artdude543** [comment]
But to be clear this pack was designed for Sevadus. It's designed around him and how he wanted it. Nothing will change in regards to the advancements or layout. It's done to a spec which we met.

**Comment by sharktemplar** [comment]
I mean concerning the issue of SevTech not explaining things but rather assuming the player knows or can easily figure them out, that is a bit of a problem. I had plenty of instances where how to do something simple wasn't detailed anywhere in SevTech, and I find myself constantly googling things from specific mods in order to use them. Most recent example would be making stabilized metal with an arc furnace. JEI only says I needed a hardened mesh, iron, and some refined hardeners. All JEI showed was that I needed all three in the same place, but it turns out the refined hardeners needed to be in the modifiers slots along with the iron ingots. Never would I have guessed that iron ingots were NOT what was being melted down to make stabilized metal, and instead were put in the modifiers slots instead of the smelting spots. I spent over an hour and a half pulling my hair out. Google was of little help this time because the recipe is different in SevTech than what it showed elsewhere and I had to eventually just rummage through random youtube videos concerning steve's carts until I noticed one from a streamer involved an arc furnace, and surprise surprise he spent almost half an hour on the stream trying to figure out how to make stabilized metal. Only when he eventually discovered iron ingots needed to be on the right slots instead of the left in the arc furnace was I then also able to continue with it myself.

There's like, probably a dozen examples of SevTech progression-related advancements where it just says "do this thing with this mod" but it doesn't tell you how, or always give you that mod book. There's a few books from mods that you have to craft in order to try and learn about that mod. It doesn't help that SevTech starts off by giving you these books when you reach those points for free, and then stops entirely leading the player to believe there IS no book to help with that mod when in fact there is, you just need to browse JEI until you find it and craft it. 
There's a lot to complain about in terms of SevTech teaching players what to do. Not just in the first two ages either. I'm in age 4 and pausing the game to go spend half an hour googling information that isn't obvious or clear to me in order to progress or acquire something exceedingly valuable is simply par for the course at this point. I've gotten used to it and I never thought about how stupid it was. It do in fact hate games that make me stop playing them in order to figure out how to continue playing them. A modpack for a game like Minecraft was probably always going to entail this, though. 

As for the 

> Let me repeat that: I am being trained, early in the game, that all that matters is advancing to the next tech level, and everything else at any given stage of the game is pointless.
> 
> You have gone to a great deal of effort to make these tech levels, put content in them, and all I am learning is to skip them as quickly as possible. That's just bad.

I actually disagree. All those extra advancements are more a quality-of-life thing. They are pointless in the same vein that the mushroom islands or woodland mansions are pointless in vanilla minecraft. If you consider defeating the Ender Dragon and watching the end credits to be the ultimate goal, then there's an ENORMOUS amount of stuff in vanilla miencraft that's pointless, and only there if you want it or find it fun to invest your time into. SevTech does this but on steroids. I've skipped kitten kaboodles of advancements because they didn't interest me. Literally never even touched Steve's Carts. Didn't even make a cart or use any of the mod at all. Only acquired stabilized metal to get a mob duplicator for my mob farm. 
I think that's fine. It allows you to select what you want to do, and only rarely forces you into mundane or tedious vague bullshit as a requirement to progress OCCASIONALLY. 
For the most part, the age-progression content is straightforward enough. There's nothing wrong with focusing on ONLY the age progressions. I for one played that way intentionally for a good while, thinking once I had enough stuff unlocked, I'd simply come back and browse what I wanted, getting it later, which is in fact now what I'm doing; building a massive sky castle that will have absolutely everything I could possibly want or need now that the majority of SevTech and its mod contents are available to me. 
If you were hoping for progression where you only advance after doing 100% of everything in that age, I would argue you don't know what you're asking for, because during age 0 and 1, I thought that's what I had to do (again because SevTech never told me that age progression advancements had a unique shape to their advancement icons) and it was a tedious nightmare doing this and finding that and completing something I cared nothing for just to try and continue. 

Overall SevTech isn't really good for player's who aren't already at least relatively familiar with a large handful of mods. It requires homework outside of the game in order to keep going further, yes. I don't like it much either in that manner. It's more a serious of mods strung together with the idea of one unlocking the next, rather than something that was delicately crafted to provide the player the complete and total experience from several of the best mods available in the pack in as smooth and convenient a way as possible.

**Comment by Seegras** [comment]
In my opinion, PrimalCore and Immersive Craft are mostly meant (and to some extent Embers, Immersive Tech, Blood Magic -- basically anything without a GUI) to be used with VR goggles. What otherwise is tedious to use becomes the more agreeable way with VR.

**Comment by keybounce** [comment]
I deliberately waited a week before replying, because I wanted to make sure that I had a chance to look at this from a point of calmness.

> This was posted due to the fact you opened a tonne of petty issues on that day all of which lack details and you expected us to help/give support on little to now information. That and most of them are Discord issues. This tracker is for bug reports and mod suggestions not.

Every place that takes bug reports says, break your bug report up. One bug report per issue. Lots of tiny reports. A big report is hard to work with, because no-one wants to start it, because it will take forever to complete. Lots of tiny reports.

Were my reports actual "help, I need support, how do I solve this"? No. They were "this is broken.".

Mipmaps causing problems. Identified in a Forge pull request as mods doing the equivalent of a graphic state push, alteration, and no balancing graphics state pop. A Forge pull request to catch it and fix it by Forge so that you don't need to worry about fixing every single mod that's messing up. Definitely a mod bug, but no clear idea which mod was messing up. Now fixable with a Forge upgrade. (Entirely separate: we are warned not to try to upgrade Forge ourself in this pack, because that will break something else).

Flint not flaking every single time. Turns out this is apparently expected. Not every attempt to make a flake gets a roll, and you won't always succeed at your attempt. So you have to attempt multiple times, and you're going to have losses. This is not documented, but is expected behavior.

A complaint about how items that you pick up off the ground go into your hot bar, and then get dropped as you move your mouse wheel. "Works as intended".

I'm getting advancements that I haven't earned. "Eh, we can't fix it".

The hiding of things that you can't detect failing in the case of ore samples. "Known, but we can't fix it". Oh, not closed as duplicate, not closed as can't fix, not closed as won't fix, but as "invalid".

An issue where a new player does not even see the work stump, because of how the advancements are laid out. Along with the comment of "Okay, now I see it". Closed without comment.

A follow-up report where I say, "the flint pickaxe trio shows up well before the work stump, even though it cannot be made until after the work stump. This is confusing.". A response of basically, "we can't do anything about this, this is how the person responsible wants it".

Another report about the Spear and Tomahawk being oddly placed in the achievements, and asking for flaked flint points to be made a tutorial-level item. Closed without comment.

Reporting an inconsistency in the way that the spear and tomahawk display information compared to everything else. "As intended".

2 different reports about problems making use of just enough items features with work stumps. "You just don't. You have to deserve it."

A report about getting information on Galactic craft at age 0, instead of something more appropriate (I'm guessing age 4). Closed without comment.

A complaint about having to pulse attack to make planks. "You tap attack on the chopping block..... stop wasting our time with these issues use discord....". And in this thread, you made it clear that you thought I was asking for support. Not reporting problems.

Please take a look at what I have written here tonight. Do you see how a new player, who has never seen this pack before, and is trying to play it through, is going to get very discouraged very quickly?

"That and most of them are Discord issues. This tracker is for bug reports and mod suggestions" -- WHICH EVERY SINGLE ONE OF THESE IS.

Discord is a horrible support tool.

Compared to forums, Discord is a horrible support tool.

There is effectively no history. There is no way to have any type of ongoing discussion about a topic. There is no way for people to comment on the same topic a day later, and be read and understood by somebody a week later.

There is next to no "pinned topic" feature like there is in forums. You can pin single messages, but there's no way to respond to them.

There is zero expectation that messages posted in Discord will be seen by anybody with the ability to deal with it. High-volume, high noise level, low signal-to-noise value, no organization, and nothing keeping track of what an administrator has/has not yet read, at least nowhere near the degree to which a forum does.

"We're not going to waste our own time when working on the pack and then deal with issues which no effort was put into them or could have been dealt with in Disord." No effort?

No Effort?

NO EFFORT?

Every single issue that I reported, excluding only the flint flaking, I mentioned how to work around the problem that I found. With flint flaking, there was no workaround because the only problem was the documentation.

You complain that I don't put any time into this? I do. You complain that I don't make any effort to solve the problems on my own? I do.

More than anything else, it is this dismissal that infuriates me.

But none of this is the real problem.

Let me restate that.

TL; DR: none of this is the real problem.

"But to be clear this pack was designed for Sevadus. It's designed around him and how he wanted it. Nothing will change in regards to the advancements or layout. It's done to a spec which we met."

This is the real issue.

Do you not see it? Then let me spell it out.

Sevadus is the one who has to approve changes. That means any issues with the modpack ultimately need to go to him. Not to you.

Trying to report any issue to you is pointless; the only one that matters is somebody who isn't even here, doesn't read these reports, doesn't get reports sent upstream to him from you for his comments, and is completely out of the loop.

So why is this issues thing even here?

Why do you have an issues section on this GitHub, when ultimately you can't actually do anything? When you just want us to use discord?

" It doesn't help that SevTech starts off by giving you these books when you reach those points for free, and then stops entirely leading the player to believe there IS no book ...". Well, no: apparently, the first person on a server to reach an age where there is a book will get the books for free. Other players don't, and if they aren't aware that the mod has a book, they are clueless. Even worse, imagine seeing a whole bunch of new things show up in your achievements, and not seeing anything at all about backpacks. Absolutely no clue that backpacks have opened up for you. In a modpack that basically triples the number of items that there are to collect. That you need. When your chest/storage space ability has been crippled.

"I'm in age 4 and pausing the game to go spend half an hour googling information that isn't obvious or clear to me in order to progress or acquire something exceedingly valuable is simply par for the course at this point. I've gotten used to it and I never thought about how stupid it was. ". Isn't this basically the point of a quest line? Sorry, an achievement line?

I was ready to give up on this completely. My friends basically apologized for rushing through, and rushing ahead so quickly. One of them got to age 3 in a solo world, (during her time off from medical school), and introduced this pack to the rest of us. Another got to age 2 in a solo world in something like 10 days. (As he says, he has no life). The 2 of them worked together with the 3rd person, and they all got at least half way through age 0 during the first week of part-time play while I was still trying to connect my own machine.

I tried to progress on my own, trying to catch up with them.

By the next weekend? The one with no life had split off from the rest of the group, made his own base, and advanced to age 1. I was struggling with basics. By the end of that weekend, no life was up to age 2, everyone else was up to age 1, we had a teleport network built between our bases, I was trying to make string (and at this point I don't even remember what it was that I needed the string for), and was trying to put down enough torches to safely light up our area, only to realize that instead of one block of wood making 8 sticks making 32 torches, one block of wood might make 6 sticks which makes 2 torches. And a mass of giant trees took more than 3 work blades to de-leaf (because you get more sticks from work blade de-leafing than you do from decay), absolute full of pain and anguish, taking a massive amount of time, and only later do we discover that there's apparently a tool in age 0 that takes the leaves off of a large area of trees in almost no time.

If it was just me, I'd already be gone. My friends have convinced me that the pack isn't quite so bad if you have someone to lead you through it, and the other comments in this thread tell me that other people need to be led through it. So the quest line/achievement lines are not sufficient, and the person who made this pack is happy with that. That stinks.

I'll probably make videos of my play through and learning, with the goal of being a "how-to" for the people who come after me, because this pack badly needs it.

**Comment by sharktemplar** [comment]
Well I mean... 
`absolute full of pain and anguish, taking a massive amount of time`
Yeah, that's SevTech for you. Even I eventually heard that the modpack had an enormously prominent tendency to be grindy, sloggish and very very painfully gradual in making progress... especially if you didn't know anything about the mods beforehand which SevTech makes you use in order to make progress. 
But the
`and only later do we discover that there's apparently a tool in age 0 that takes the leaves off of a large area of trees in almost no time.`
Eeeeeyup. Definitely everything SevTech is about. It's a Minecraft-themed Homework Simulator. 
When I was getting through age 3, I was making a bee-line for the age advancements to breach age 4, hoping blindly it would unlock diamonds by age 4 because why in the world would diamonds be exclusive to an age that prioritizes space travel? How are diamonds more valuable than literal rocket-science? 
I wanted them because in the Cyclic book, I read about diamond spikes, which were the first item I discovered that apparently would act as a player-kill. Already having a small, rather inefficient mob farm built by that point, all I wanted were those diamond spikes so I could turn my mob farm into an experience farm as well.
Come age 4 after some grueling and nightmare-ish progression grinding and googling and video watching and building and rebuilding when I realize I built it wrong and all the hullabaloo that comes with stupidly vague SevTech progression... I discover diamonds are in fact age 5 not 4, but browsing through JEI, I noticed mob grinding utils is now unlocked... which comes with mob-killing items that drop experience, and dark utilities now has a "player trap" which does the same thing as diamond spikes. 
My ultimate goal was wisped away from me due to lack of SevTech information, and then immediately returned to me through another avenue via lack of SevTech information. 
This is unfortunately how the whole experience is, honestly. You will have a suckish time if you don't constantly browse and research what is newly available. 

SevTech seriously seriously VERY EXTREMELY BADLY **NEEDS** a feature where it lists to you all the items unlocked by age, and also provide links to resources players will end up likely googling anyway just to try and learn more about a mod they are using.
Since SevTech has so many mods, it needs to compensate for this massive overflowing incline of a learning curve it now results in, and provide players something more comprehensive. 
Something to list what each age unlocks (only available after you reach that specific age).
Way more detail about where to find more information concerning a mod (I.E. does the mod have a book the player can craft? Or should they default to googling and thus be provided a link from an in-game description instead?). 

SevTech is very anti-player-friendly. 
Though to be honest, apathetic-looking github management and poor/nonexistent in-game teaching methods aside...
I just want the damn modpack to work again. Seriously, whatever happened with the 3.0.8 update has made it so incredibly unfun to play SevTech with how unstable it is now that I may not ever have the motivation to actually make it to age 5. 
Literally a complete rollback to 3.0.7 would be 100% acceptable to me at this point, because at least the modpack can be played in a heavy setting again, such as large builds or complicated surroundings (which, you know... age 4 and 5 will basically _demand_ from the player anyway). 

Unfortunately, we may just be barking at a bush from the perspective of the people involved in sevtech development. If it was made primarily to meet a specific guy's standards, I doubt anybody but him coming forward and saying "shit needs a lot of work, yo" is gonna do anything besides fall on deaf ears.

**Comment by Buckmanfu** [comment]
The 'poor me' that these posts are dripping with is infuriating, since it would unprofessional for the Dev Team to excoriate you, I'll step up. 

Most of this complaining is stupid.  You're complaining about lack of documentation, that is the Forge/Modded world in general.  Very few mods have 'user level' documentation, most tutorials that I have seen, have been from people taking the time to either have in-depth QA with the author, reading the API, or just experimenting.  

If you want your hand held through a puzzle pack, find a new hobby, this isn't for you.   Seriously, at worst SevTech is breaking your ego because you have forgotten what it's like to not know something about MineCraft.  I just started Modded this past Winter, as such, with every new pack there are different mods for me to learn.  SevTech merely took every thing you thought you knew and turned it on it's head, and you are butt hurt because you feel like a noob, and you're acting like a child, grow up.

This was a commissioned work.  I am glad that the Dev Team and Sevadus have made this work public, it's challenging with many paths to achieve overall completion.  I am having fun with it.  It is frustrating with some of the changes to mechanics that are not apparent, but the Discord community is extremely helpful and courteous. 

Most of the issues you have brought up, are your issues.  There has not been anything in this pack that I have not found information on by watching Streamers, YouTubers (both mod tutorials and SevTech LP), or by reading/asking on Discord. 

It might simply be this mod pack is not for you, but that is all on you not the Dev Team.  There are 1200+ other mods and 2000+ video games released every year, and if that's still not for you, Crayola might have something more your speed.

**Comment by keybounce** [comment]
Sorry, no.

Number one: this is a pack with custom recipes, and custom unlocks. It's not just "look up the docs for a mod". It's "trying to figure out what was changed specifically for this pack".

Number 2: Again: requiring Discord to have any clue about what you're doing is the wrong idea.

Number 3: Please re-read what you just wrote. Assume that that was written to you, instead of by you. Would you consider that to be insulting? I did.

Number 4: You are not Sevadus. We have already established that this pack is one person's view, as implemented by a team. Not you. You do not have the authority to make the declarations you just made. They are your opinions only, yet you want to treat them as absolute truths.

Please re-read what I said after my TL DR.

>  You're complaining about lack of documentation, that is the Forge/Modded world in general. Very few mods have 'user level' documentation, most tutorials that I have seen, have been from people taking the time to either have in-depth QA with the author, reading the API, or just experimenting.

You have just described the mods that I don't use or play with. If I am looking at potentially hundreds of mods, I am not going to spend 30 minutes to 2 hours on each one evaluating it. Mods that I play with are mods where the forum post explains enough of what the mod does/how it does it, and if it has a wiki, it's not just an empty skeleton for the users to populate, but an actual info dump from the author of the mod.

Conversion modpacks? I've played a couple. My minimum standard is around the level of agrarian skies. That wasn't perfect, but it was good enough. Resource limited maps like skyblock in vanilla Minecraft make you think, but you're working with what you know. Maps where you start with essentially one sapling, some useless ground, and a bunch of mods you might have never played with, and then say "go study these mods for another 3 hours; we won't tell you which mods to look at, we won't tell you what your goal is, and your only clue is to come from someone else who has already gone through this and figured it out" are nothing more than distributed time wasting.

This pack is right in the middle. It is distributed time wasting, but hey, at least we were clearly pointed at totemic upon entering age 0. Except for the things that we weren't pointed at, like absolutely no hint that there are backpacks in this pack.

This is not "poor me".

This is "I think this is bad design, and here is why."

The response is basically "only one person has the authority to say I agree, or I disagree, and that person isn't here".

**Comment by Buckmanfu** [comment]
JEI has literally EVERY RECIPE in the pack.... so whats the problem?   It's even setup in a manner where, if it's not in JEI, you can't make it.... yet. 

No one said Discord was required. 

I offended you? Working as intended.  Your OP is offensive given that you paid exactly nothing for SevTech but have the gall to stand up on a soap box and complain about, how a group of hobbyists did not make this pack for the general population.  As you have repeated pointed out, the pack was made for Sevadus.  Since you haven't paid money for the pack,  they are under no obligation to do anything.  Yet your attitude is that of someone that paid money for a product and is expecting a much higher tier of support.   This make you and entitled jerk.  


I have perused all of your tickets... honestly most of them are just a difference of opinion, and you, in this post come off as your opinion should have more weight than it really does.   The remainder are issues with the behavior of a specific mod which would need to be taken up with the mod author, not the pack author.   Based on some of the responses... you need to learn how to use the search/filter button. 

This issue tracker is for things not working as the AUTHOR intended, not how you think it should be, nor how you would have done it. 

You also mention that you're playing as part of a group... why are you not using the group mechanics? 


This pack has a theme, a structure that was intended from the outset.  Your opinion on implementation does not matter in the slightest.  You were not the intended audience.

**Comment by sharktemplar** [comment]
@Buckmanfu
Your first comment starts with the bitching that none of us with complaints about SevTech's design have any merit because "that's just the way it is", which is nonsensical reasoning akin to "it's okay if somebody murders your dog because that's just the way it is". 
Lack of documentation being present in the forge/modded world is not an excuse for lack of documentation. It's still lack of documentation which comes at the expense of the player. A modpack like SevTech which has so many different mods that need to be understood in order to progress through the modpack takes this problem and bloats it up tenfold, thus rendering it a perfectly valid complaint that SevTech has a problem with not providing enough information to the player compared to how much is being asked of the player. 
You literally admit it's frustrating to progress, but your solution that excuses this is to consult the discord, rummage through youtube videos and streams, and google what you don't know...
That's literally what I said was par for the course in SevTech, and thus a problem with SevTech. It's not some hidden justification you're reeling out from between the cushions.
It's **ALWAYS** stupid design to force a player to stop playing in order to figure out how to continue playing. Hence why I called it a Minecraft-themed Homework Simulator. 
I'm well aware SevTech isn't responsible for the existence of all these other mods that need to be individually learned and then used, but SevTech _is_ responsible for incorporating them, and then providing either a bear minimum, an inconsistent degree of, or NO information/resource directions at all, thus making it perfectly valid to complain about it. You're making an excuse for SevTech Ages that doesn't make sense and it only comes off as childish white-knighting for your favorite modpack.
Do you think we're attacking SevTech because we hate it?
Lemme clue you in since you seem hopped up on how much of a voice of reason you _think_ you are being; we enjoy SevTech a great deal. If we didn't, we never would go to so much effort to try and pinpoint what needs to better in SevTech. Nobody who hates something will spend collective hours rummaging through its problems or making highly detailed reports about said problems. They will dismiss it out of spite for it and move on. Got it? 
Your thoughtless ultimatum of "there's SO MANY OTHER GAMES, JUST play those!" does not excuse any problems we've brought forth. It's literally just saying "don't like what's wrong? leave!" which is a terrible way for something to improve if every single time you're met with a critic, you show that critic the door and pretend nothing they say matters just because you don't like it. 

What if your favorite game had a serious problem where it crashed to desktop every time you booted up the main menu? You go to the troubleshooting forums for that game, and the developers say "there's 2000+ other games released every year. Try one of those. Bye!" in response to your complaint of that issue.
Do you not see how idiotic of a practice/excuse that is when it comes to improving something? 
Also...

> JEI has literally EVERY RECIPE in the pack.... so whats the problem? It's even setup in a manner where, if it's not in JEI, you can't make it.... yet.

lol no. First of all, there's plenty of things that aren't even inputted into JEI when they should be, hence why the last 2 updates added things to JEI which were already present, but didn't show up in JEI. An example of this is the buckets. In 3.0.6, typing in bucket would show only clay buckets even if you had iron buckets unlocked, and it also would not show all the possible liquid types that can be placed in a bucket (and still doesn't do this if I recall correctly). 
Not to mention that JEI updates recipes without telling the player. I was making wooden gears with teeth for WAY longer than I should have, because I didn't realize the recipe for wooden gears had updated to a much simpler one now that I had a mechanically powered saw to cut wood. I only realized this after watching someone else playing SevTech who pointed it out directly, and they only pointed it out directly because it's a common thing to overlook which results in a lot of time and resources wasted by the player early-game. If you didn't realize a recipe had updated from what you were already taught that it was due to unlocking a new age, what should you as the player do? I mean what is your retarded justification for _that_? Are you actually going to imply that a player should look through every single recipe they've already used just to check if it has an updated and easier one EVERY single time they breach a new age? 
Don't be ridiculous just because you confuse critique with personal attacks. SevTech **NEEDS** something that at least makes a tiny little _attempt_ to inform the player of something new that has occurred, or what is expected of them with it. Be that a composed list of all the changes a new age brings that they can browse through at their leisure, much like a JEI list except only showing specific age-unlocked recipes/items, or more descriptive information about how a player is to engage with a certain block if that is the only block they need from a mod, or provide links to resources where the player can learn more about that mod a little quicker than literally halting gameplay entirely to watch a video or stream in order to figure out one single thing. 

> No one said Discord was required.

You literally just did in your previous comment. It was part of your argument of how SevTech isn't difficult to figure out because one can consult the discord, or go watch youtube videos/streams in order to learn something, which again, is a very very stupid proposition that literally encourages stopping gameplay in ALL its entirety in order to figure out how to continue with progression in ANY way. 

> Your OP is offensive given that you paid exactly nothing for SevTech but have the gall to stand up on a soap box and complain about, how a group of hobbyists did not make this pack for the general population.

You are not required to personally approve of the way somebody words their OP.
SevTech being free to play so long as you own a legal copy of Minecraft is not an excuse to dismiss problems or complaints. The Issues section of the SevTech github exists for a reason. 
Just because something is free doesn't mean everybody magically now has NO right to complain about any portion of it in any way, ever. That's not how it works. Again, people only complain in detail about something they care for. We care about and enjoy SevTech, thus we exert the effort to provide plentiful detail in what we do not like about SevTech. If we were posting "lol it's gay fuck off" as a github issue, you would have a flawless argument to stand on. This however, is very much not the case. 

> The remainder are issues with the behavior of a specific mod which would need to be taken up with the mod author, not the pack author. 

This is yet _another_ thing you've said that screams how you have no clue how any of this actually is supposed to work. 
No, we are not supposed to go to the mod author first. We are supposed to go to the modpack author first, because the modpack author may not be using the most recent version of a mod, or the issue in that mod may be caused by a conflict with another mod in said modpack. So if people did it your way, (and this has happened plenty before, thus why mod authors such as the Thaumcraft fellow would call you dumb for your suggestion) then people would be flocking to a mod author's github or mod page complaining about problems that have already been fixed, or problems that cannot be repeated in the standalone mod but only occur in said modpack those users are experiencing those problems in, and thus are useless spam to that mod author, because there's nothing the mod author can do except mindlesly track a bug that doesn't exist because it has a good chance of only existing when applied in somebody else's modpack, or is straight up from an older version of the mod that the mod author fixed long ago, but the modpack never kept up.
I'm trying to explain this as clearly to you as possible. Do you see how stupid it would be to go to the mod author about a problem with a mod you experienced while playing a modpack? 

> 
This issue tracker is for things not working as the AUTHOR intended, not how you think it should be, nor how you would have done it.

Again... oh my God... you're gloriously incorrect. 
Tell me... what do you think this means?
https://i.imgur.com/VM23MMI.png
When you click "New Issue" in the issues section of the github, you get two options. One for a bug (which this post is not formatted under, thus proving it's not about a bug) and one for feature requests **which include changes to existing aspects of the modpack**. 
There's already plenty of mods in SevTech Ages that have been tweaked, or had recipes added/removed exclusively to better fit the SevTech experience, so your argument that mod features in a mod pack should only be brought up if they are not working as that specific mod author intended is flat-out 
**W. R. O. N. G.** 

> You also mention that you're playing as part of a group... why are you not using the group mechanics?

Because group mechanics are buggy and inconsistent. Hell, even watching a popular SevTech Let's Play (which you took as such a natural thing to do when getting stuck in SevTech) such as captain sparklez will provide plentiful evidence of this. Advancements not being given to one person in the same team but being given to another, seemingly breaking in this way at complete random, and almost always requiring a force-sync. Can't blame somebody for not relying on a mechanic that almost never works as intended. 

> This pack has a theme, a structure that was intended from the outset. Your opinion on implementation does not matter in the slightest. You were not the intended audience.

Yes, it has a theme. A cool one at that. Yes, it has a structure. A shaky one at that. No, the opinion of the player of a modpack matters quite a bit. Your fallacious argument that none of us are Sevadus means none of what we say matters in any way no matter what, does NOT hold up as an argument, because SevTech was released to the public, and had a github opened specifically for tracking issues and offering feature requests. The mere existence of this exact github you are commenting your asinine nonsense on is proof that the SevTech devs do actually care what other people think at least to _some_ degree, and absolutely want to hear about it. If they didn't want to hear about it, and didn't care what others thought, and only took issues into account if Sevadus himself was the one complaining, they wouldn't have opened a public github for SevTech that anybody who plays SevTech could access **literally from the main menu of SevTech Ages**. 
We very much are the intended audience because of how and why SevTech Ages was launched as a modpack. You do realize your own point could be turned against you in a heartbeat, right?
If our opinion doesn't matter, your opinion of our opinion doesn't matter because you aren't Sevadus either. You aren't who commissioned SevTech to be developed either. You aren't the target demographic either, and thus nothing you say that is even remotely pertaining to SevTech Ages (including what you say about other people who play SevTech Ages) has any weight what-so-ever because you aren't Sevadus.

Do you see how stupid that is yet?

Anyway, I had find dissecting your bs. Next time don't try so hard to discredit other people's complaints simply because you have lower standards for what a Minecraft modpack offers in its functionality. 

_**You are not required to approve of other people's complaints in order for those complaints to qualify as valid.**_

**Comment by artdude543** [comment]
Okay, that's quite enough of this. This has gone well out of proportion and you guys are now at each other's throats.
